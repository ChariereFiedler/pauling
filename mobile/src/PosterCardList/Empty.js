import React from 'react';

import { View } from 'react-native';
import { Icon, Text } from 'native-base';

import styles from './styles';


const Empty = () => (
  <View style={styles.Empty}>
    <Icon
      name="md-qr-scanner"
      style={styles.MessageIcon}
    />
    <Text style={styles.Message}>
      Scan a Pauling QR code to add a first poster to your collection
    </Text>
  </View>
);

export default Empty;
