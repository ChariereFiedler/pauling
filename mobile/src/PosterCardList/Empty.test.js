import React from 'react';
import { shallow } from 'enzyme';

import Empty from 'app/PosterCardList/Empty';

describe(__filename, () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Empty />);
    expect(wrapper).toMatchSnapshot();
  });
});
