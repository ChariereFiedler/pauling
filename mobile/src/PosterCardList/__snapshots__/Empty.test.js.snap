// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty renders correctly 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-end",
    }
  }
>
  <Styled(Icon)
    name="md-qr-scanner"
    style={
      Object {
        "color": "#777",
        "fontSize": 80,
        "paddingTop": 50,
      }
    }
  />
  <Styled(Text)
    style={
      Object {
        "color": "#333",
        "fontSize": 24,
        "paddingLeft": 20,
        "paddingRight": 20,
        "paddingTop": 50,
        "textAlign": "center",
      }
    }
  >
    Hi! I am Pauling, nice to meet you.
  </Styled(Text)>
  <Styled(Text)
    style={
      Object {
        "color": "#777",
        "fontSize": 20,
        "paddingLeft": 20,
        "paddingRight": 20,
        "paddingTop": 50,
        "textAlign": "center",
      }
    }
  >
    You can add a poster to your collection by scanning a Pauling QR code. Use the button below to open your camera and follow the instructions.
  </Styled(Text)>
</Component>
`;
